packages/jest-parallel/src/types.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+// Because of the dynamic nature of a worker communication process, all messages
+// coming from any of the other processes cannot be typed. Thus, many types
+// include "any" as a flow type, which is (unfortunately) correct here.
+
+/* eslint-disable no-unclear-flowtypes */
+
+export type JestApi = {[string]: (...any) => Promise<any>};
```

Comment:
Can you drop "Jest" from the types everywhere? I don't really think that adds much.